.vscode/launch.json

```diff
@@ -4,6 +4,17 @@
   // For more information, visit: https://go.microsoft.com/fwlink/?linkid=830387
   "version": "0.2.0",
   "configurations": [
+    {
+      "name": "Launch Extension",
+      "type": "extensionHost",
+      "request": "launch",
+      "runtimeExecutable": "${execPath}", // open another instance of vs code
+      "args": ["--extensionDevelopmentPath=${workspaceRoot}/vscode-extension"],
+      "outFiles": ["${workspaceRoot}/vscode-extension/client/out/**/*.js"],
+      "autoAttachChildProcesses": true,
+      "preLaunchTask": { "type": "npm", "script": "watch" },
+      "internalConsoleOptions": "openOnSessionStart",
+    },
```

Comment:
when I'm working on this, I set this launch config as my default "Run and Debug" option in VS code, so I can F5 to test the extension